### DISCUSSION TITLE
A Repeater feature for editing and re-sending requests which were "received" but not "intercepted"

### DISCUSSION THREAD

**Discussion by payloadartist** [google/tamperchrome#193] [Ideas]
It would be great to have a feature than can allow editing and playback of requests which were **Received** (not in **Intercepting mode**)

Sometimes we need to go back to certain requests recorded in the requests history to test out an idea. So, would be great if there was a feature to **edit and repeat** received requests (outside Intercepting mode)

Not sure if there's a way to edit and send previous requests currently as no Send option doesn't appear

Thanks!

**Comment by sirdarckcat** [comment]
Yep, this is something that is definitely useful and was part of the original mocks in #18 but hasn't been implemented yet as to first nail-down the UX for interception, and because there's a lot of extensions that provide very good support for log-and-replay.

Anyway, follow this on #195 :)
